### ISSUE TITLE
UnnamedParameterUse cannot suppress constructors

### ISSUE THREAD

**Issue by matejdro** [detekt/detekt#9030]
## Expected Behavior

Adding a class name to the `UnnamedParameterUse.ignoreFunctionCall` should suppress this rule for the constructor calls to this class.

## Observed Behavior

`UnnamedParameterUse` still gets triggered.

## Steps to Reproduce

1. Set the following config:

```yaml
potential-bugs:
  UnnamedParameterUse:
    active: true
    ignoreFunctionCall:
      - com.mypackage.Test
```

2. Run detekt on the following code

```kotlin
package com.mypackage

data class Test(
    val first: Int,
    val second: Int
)

fun run() {
    Test(1, 2)
}
```

## Your Environment

* Version of detekt used: 2.0.0-alpha.2
* Version of Gradle used (if applicable): 9.2.1

**Comment by atulgpt** [comment]
You need to use `"com.mypackage.Test.<init>"` for suppressing the constructor

Below TC is passing
```
@Test
fun `TC with ctor call`() {
    val code = """
        package com.mypackage
        class Test(a: Int, b: Int, c: Int)
        fun foo() {
            Test(1, 2, 3)
        }
    """.trimIndent()
    val findings = getSubject(
        ignoredFunctionCalls = listOf("com.mypackage.Test.<init>")
    ).lintWithContext(env, code)
    assertThat(findings).isEmpty()
}
```

**Comment by matejdro** [comment]
Ah, thanks. Do you think documentation could be updated to specify that?

**Comment by atulgpt** [comment]
There is one issue https://github.com/detekt/detekt/issues/8783 and I have raised a [PR](https://github.com/detekt/detekt/pull/9053) which commonize the documentation so it should be fixed after that PR

**Comment by matejdro** [comment]
Thank you! Closing this ticket then.
